<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1800</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QColumnView" name="columnView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Trong nhà</string>
    </property>
   </widget>
   <widget class="QColumnView" name="columnView_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>521</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ngoài sân</string>
    </property>
   </widget>
   <widget class="QColumnView" name="columnChooseView">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>30</y>
      <width>441</width>
      <height>831</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFood">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>50</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Đồ ăn</string>
    </property>
    <property name="icon">
     <iconset theme="phone">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDrink">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Đồ uống</string>
    </property>
   </widget>
   <widget class="QColumnView" name="columnDisplayView">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>771</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QColumnView" name="columnPaymentView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>511</y>
      <width>1301</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên khách hàng</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>650</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tiền mặt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Thẻ tín dụng</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>70</y>
      <width>37</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thanh toán</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lầu 2</string>
    </property>
   </widget>
   <widget class="QColumnView" name="columnView_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>521</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>550</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tổng thành tiền</string>
    </property>
   </widget>
   <widget class="QLabel" name="tongTienLabel">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>550</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>590</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Chiết khấu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>650</y>
      <width>37</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VAT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>700</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tổng cộng</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>680</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="thanhToanButton">
    <property name="geometry">
     <rect>
      <x>1109</x>
      <y>747</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Thanh toán</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>750</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Tạo mới</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>700</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ghi chú</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>700</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>600</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã giảm giá</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>110</y>
      <width>401</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Download/5.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="traSuaChoose">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#traSuaChoose{
background-color: transparent;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="kemCheeseChoose">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#traSuaChoose{
background-color: transparent;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="traDaoChoose">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#traSuaChoose{
background-color: transparent;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>809</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>809</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="hoaDon">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>41</y>
      <width>741</width>
      <height>441</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>184</number>
    </attribute>
    <column>
     <property name="text">
      <string>Tên món hàng</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Số lượng</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Đơn giá</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thành tiền</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>650</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10%</string>
    </property>
   </widget>
   <widget class="QLabel" name="tongTienCuoiLabel">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>700</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>columnView_6</zorder>
   <zorder>columnView</zorder>
   <zorder>label</zorder>
   <zorder>columnView_2</zorder>
   <zorder>label_2</zorder>
   <zorder>columnChooseView</zorder>
   <zorder>pushButtonFood</zorder>
   <zorder>pushButtonDrink</zorder>
   <zorder>columnDisplayView</zorder>
   <zorder>columnPaymentView</zorder>
   <zorder>label_7</zorder>
   <zorder>textEdit</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>tongTienLabel</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>label_15</zorder>
   <zorder>line</zorder>
   <zorder>thanhToanButton</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>label_16</zorder>
   <zorder>textEdit_2</zorder>
   <zorder>textEdit_3</zorder>
   <zorder>label_17</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>addButton</zorder>
   <zorder>minusButton</zorder>
   <zorder>hoaDon</zorder>
   <zorder>label_3</zorder>
   <zorder>tongTienCuoiLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
